pkgs/applications/science/electronics/gnucap/default.nix

```diff
@@ -9,6 +9,10 @@ stdenv.mkDerivation rec {
     sha256 = "12rlwd4mfc54qq1wrx5k8qk578xls5z4isf94ybkf2z6qxk4mhnj";
   };
 
+  makeFlags = [
+    "TARGET_EXT=${stdenv.hostPlatform.extensions.sharedLibrary}"
```

Comment:
With this change the package doesn't seem to build anymore for me on linux, with:

```
make[2]: Leaving directory '/build/gnucap-20210107/lib/O'
make[1]: Leaving directory '/build/gnucap-20210107/lib'
(cd modelgen; make install)
make[1]: Entering directory '/build/gnucap-20210107/modelgen'
make default
make[2]: Entering directory '/build/gnucap-20210107/modelgen'
mkdir -p O
cat Make1 Make2 ../Make3 Make.depend >O/Makefile
(cd O; make -k)
make[3]: Entering directory '/build/gnucap-20210107/modelgen/O'
make[3]: 'gnucap-modelgen' is up to date.
make[3]: Leaving directory '/build/gnucap-20210107/modelgen/O'
make[2]: Leaving directory '/build/gnucap-20210107/modelgen'
(cd O; make install)
make[2]: Entering directory '/build/gnucap-20210107/modelgen/O'
make[2]: *** No rule to make target 'gnucap-modelgen.so', needed by 'install'.  Stop.
make[2]: Leaving directory '/build/gnucap-20210107/modelgen/O'
make[1]: *** [Makefile:79: install] Error 2
make[1]: Leaving directory '/build/gnucap-20210107/modelgen'
make: *** [Makefile:63: install] Error 2
builder for '/nix/store/99rzw0pr2k9n67qi84cidal5qqqx6gnb-gnucap-20210107.drv' failed with exit code 2
```

Reply:
Oh duh. I didn't mind the empty `TARGET_EXT = ` lines in the makefiles, but of course the purpose of those is executables. Any way, I removed it and just kept the broken flag.